package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "hacksoc.org",
+  "version": "0.1.0",
+  "description": "Replacement generator for static pages on HackSoc.org",
+  "main": "main.js",
+  "scripts": {
+    "mkdir": "mkdir -p minutes html"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/HackSoc/hacksoc.org.git"
+  },
+  "author": "Luke Moll",
+  "license": "ISC",
```

Comment:
We need to decide on a license for the website generator (my vote, as always, is for BSD-3-Clause).

Reply:
I think I'd be happy with BSD-3-Clause